docs/CHANGELOG.asciidoc

```diff
@@ -82,6 +83,140 @@ Review important information about the {kib} 8.x releases.
 include::upgrade-notes.asciidoc[]
 
 
+[[release-notes-8.17.0]]
+== {kib} 8.17.0
+
+coming::[8.17.0]
+
+For information about the {kib} 8.17.0 release, review the following information.
+
+[float]
+[[deprecations-8.17.0]]
+=== Deprecations
+
+The following functionality is deprecated in 8.17.0, and will be removed in 9.0.0.
+Deprecated functionality does not have an immediate impact on your application, but we strongly recommend
+you make the necessary updates after you upgrade to 8.17.0.
+
+      
+[discrete]
+[[deprecation-197227]]
+.Removes `metrics:allowCheckingForFailedShards` advanced setting.
+[%collapsible]
+====
+*Details* +
+The `metrics:allowCheckingForFailedShards` advanced setting has been removed. With this change, it is no longer possible to suppress warnings about failed shards in TSVB. For more information, refer to ({kibana-pull}197227[#197227]).
+====
+      
+[float]
+[[features-8.17.0]]
+=== Features
+{kib} 8.17.0 adds the following new and notable features.
+
+Alerting::
+* The Case action is now GA ({kibana-pull}196972[#196972]).
+* Files can now be attached to cases directly via API ({kibana-pull}198377[#198377]).
```

Comment:
Let's have a dedicated section for Cases.

Reply:
Thanks @cnasikas! I've added the issues you mentioned and created a dedicated section for Cases.